homework3/task02/task_02.py

```diff
@@ -0,0 +1,22 @@
+import hashlib
+import multiprocessing
+import random
+import struct
+import time
+
+
+def slow_calculate(value):
+    """Some weird voodoo magic calculations"""
+    time.sleep(random.randint(1, 3))
+    data = hashlib.md5(str(value).encode()).digest()
+    return sum(struct.unpack("<" + "B" * len(data), data))
+
+
+def pool_handler(data: int):
+    """
+    Function that use multiprocessing for faster output of slow_calculate func
+    :param data: input value for calculating
+    :return: None
+    """
+    p = multiprocessing.Pool(20)
+    p.map(slow_calculate, data)
```

Comment:
the process object needs to be closed. see examples here: https://docs.python.org/3.7/library/multiprocessing.html

Reply:
in task its written:

Calculate total sum of slow_calculate() of all numbers starting from 0 to 500.

there is nothing said about "data" parameter. please, follow the task